android/quest/src/main/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireViewModel.kt

```diff
@@ -130,13 +131,16 @@ constructor(
     readOnlyLinkIds: List<String>? = emptyList()
   ): Questionnaire? =
     defaultRepository.loadResource<Questionnaire>(questionnaireConfig.id)?.apply {
-      if (questionnaireConfig.type.isReadOnly() || questionnaireConfig.type.isEditMode()) {
+      if (questionnaireConfig.type.isReadOnly()) {
```

Comment:
@ellykits , here its the main change I found to fix the underline issue, 
the required/expected behavior is handle with line 141 below.